website/yw_website/yw_website/urls.py

```diff
@@ -17,18 +17,23 @@
 from django.urls import path, include
 from django.conf import settings
 from django.conf.urls.static import static
-from script_upload import views
+from script_upload import views as script_upload_views
+from yw_rest_services import views as yw_rest_services_views
```

Comment:
perform a pull on the master branch and fix merge conflicts locally before pushing